packages/application-components/materials/media-queries.css

```diff
@@ -0,0 +1,13 @@
+/* MAX-WIDTH */
+@custom-media --screen-max-mobile (max-width: 767px); /* xsm */
+@custom-media --screen-max-desktop (max-width: 1023px); /* sm */
+@custom-media --screen-max-biggerdesktop (max-width: 1279px); /* md */
+@custom-media --screen-max-giantdesktop (max-width: 1679px); /* lg */
+@custom-media --screen-max-jumbodesktop (max-width: 1919px); /* xlg */
+
+/* MIN-WIDTH */
+@custom-media --screen-min-mobile (min-width: 768px); /* xsm */
+@custom-media --screen-min-desktop (min-width: 1024px); /* sm */
+@custom-media --screen-min-biggerdesktop (min-width: 1280px); /* md */
+@custom-media --screen-min-giantdesktop (min-width: 1680px); /* lg */
+@custom-media --screen-min-jumbodesktop (min-width: 1920px); /* xlg */
```

Comment:
Q: should we provide this in other formats as well, like we do with the custom properties? E.g. json, js...

Reply:
@emmenko good point, if we do that, we should actually use the `breakpoints` that are exposed in `ui-kit` custom properties as well. I'll look into this 🎉